tests/unit/specs/components/AboutMe/AboutMyPersonalInfo/AboutMyPersonalInfo.nuxt.spec.ts

```
@@ -0,0 +1,64 @@
+import type { mount } from "@vue/test-utils";
+import type { ComponentMountingOptions } from "@vue/test-utils/dist/mount";
+
+import { TEST_NUXT_RUNTIME_CONFIG } from "@tests/unit/utils/constants/nuxt-test.constants";
+import { mountSuspendedComponent } from "@tests/unit/utils/helpers/mount.helpers";
+
+import AboutMyPersonalInfo from "@/components/AboutMe/AboutMyPersonalInfo/AboutMyPersonalInfo.vue";
+
+describe("AboutMyPersonalInfo Component", () => {
+  let wrapper: ReturnType<typeof mount<typeof AboutMyPersonalInfo>>;
+
+  mockNuxtImport("useRuntimeConfig", () => () => TEST_NUXT_RUNTIME_CONFIG);
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)